org.adempiere.ui.zk/WEB-INF/src/web/js/org/idempiere/commons/clipboard.js

```diff
@@ -0,0 +1,12 @@
+if (typeof window.idempiere === 'undefined')
+	window.idempiere = {};
+
+window.idempiere.copyToClipboard = function(prm){
+	var tempInput = document.createElement("input");
```

Comment:
the navigator.clipboard api is pretty well supported now and we should use that instead.
see https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard and https://www.w3schools.com/howto/howto_js_copy_clipboard.asp